src/components/Recommend/Recommend.tsx

```diff
@@ -9,84 +9,47 @@ import {
   RecommendationsContainer,
   RecommendationBody,
   Icon,
-  FooterLink,
-  FooterHalf,
-  FooterWrapper,
   Column,
   RecommendationItem,
 } from './Recommend.style';
 import {
   RecommendationWithIcon,
   mainContent,
 } from 'cms-content/recommendations';
-import { HeaderWrapper } from 'components/LocationPage/ChartsHolder.style';
-import SmallShareButtons from 'components/SmallShareButtons';
-import { EventAction, EventCategory, trackEvent } from 'components/Analytics';
-import { trackRecommendationsEvent } from 'common/utils/recommend';
-import ExternalLink from 'components/ExternalLink';
+import { LocationPageSectionHeader } from 'components/LocationPage/ChartsHolder.style';
+import { HeaderWrapper } from 'components/VulnerabilitiesBlock/VulnerabilitiesBlock.style';
+import ShareButtons from 'components/SharedComponents/ShareButtons';
+import { EventCategory } from 'components/Analytics';
 import { Subtitle1 } from 'components/Typography';
 import { useBreakpoint } from 'common/hooks';
 
-const { header, footer } = mainContent;
+const { header } = mainContent;
 
-const trackShareFacebook = () =>
-  trackRecommendationsEvent(EventAction.SHARE, 'facebook');
-
-const trackShareTwitter = () =>
-  trackRecommendationsEvent(EventAction.SHARE, 'twitter');
-
-const trackCopyLink = () => {
-  trackRecommendationsEvent(EventAction.COPY_LINK, 'recommendations');
-};
-
-const Header = (props: { introCopy: string; locationName: string }) => {
-  const { introCopy, locationName } = props;
+const Header = (props: {
+  introCopy: string;
+  locationName: string;
+  shareUrl: string;
+  shareQuote: string;
+}) => {
+  const { introCopy, locationName, shareUrl, shareQuote } = props;
   return (
-    <Fragment>
+    <>
       <HeaderWrapper id="recommendations">
-        <HeaderCopy>{header}</HeaderCopy>
+        <LocationPageSectionHeader>
+          <HeaderCopy>{header}</HeaderCopy>
+        </LocationPageSectionHeader>
+        <ShareButtons
+          eventCategory={EventCategory.RECOMMENDATIONS}
+          shareUrl={shareUrl}
+          shareQuote={shareQuote}
+        ></ShareButtons>
```

Comment:
since there arent any children, you can write this as:
```
        <ShareButtons
          eventCategory={EventCategory.RECOMMENDATIONS}
          shareUrl={shareUrl}
          shareQuote={shareQuote}
        />
```